src/fastmcp/server/sampling/run.py

```
@@ -0,0 +1,291 @@
+    )
+    has_tools_capability = session.check_client_capability(
+        capability=ClientCapabilities(
+            sampling=SamplingCapability(tools=SamplingToolsCapability())
+        )
+    )
+
+    if fastmcp.sampling_handler_behavior == "always":
+        if fastmcp.sampling_handler is None:
+            raise ValueError(
+                "sampling_handler_behavior is 'always' but no handler configured"
+            )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)